index.html

```diff
@@ -889,67 +889,133 @@ <h3>Supporting algorithms</h3>
-            </li>
-            <li>
-              Let |focusedDocument| be the |topLevelBC|'s
-              <a href="https://html.spec.whatwg.org/multipage/interaction.html#currently-focused-area-of-a-top-level-browsing-context">
-                currently focused area</a>'s [=Node/node document=].
-            </li>
-            <li>
-              If |relevantGlobal|'s [=relevant settings object=]'s [=origin=] is [=same origin=] with
-              |focusedDocument|'s [=origin=], return true.
-            </li>
-            <li>
-              Otherwise, return false.
-            </li>
-          </ol>
-        </li>
-      </ol>
-      <aside class="note">
-        As there might be multiple observers, each with a different [=requested sampling rate=], the underlying
-        [=platform collector=] will need to use a [=sampling rate=] that fulfills all these requirements. This also
-        means that not every data sample from the [=platform collector=] needs to be delivered to each active
-        observer.
-      </aside>
+          </li>
+          <li>
+            [=list/For each=] |origin| in
+            <a href="https://w3c.github.io/picture-in-picture/#initiators-of-active-picture-in-picture-sessions">
+              initiators of active Picture-in-Picture sessions</a>:
+            <ol>
+              <li>
+                If |relevantGlobal|'s [=relevant settings object=]'s [=origin=] is [=same origin=] with |origin|, return true.
+              </li>
+            </ol>
+          </li>
+          <li>
+            If |relevantGlobal|'s [=browsing context=] is [=context is capturing|capturing=], return true.
+          </li>
+          <li>
+            Let |topLevelBC| be |relevantGlobal|'s [=browsing context=]'s [=top-level browsing context=].
+          </li>
+          <li>
+            If |topLevelBC| does not have [=top-level traversable/system focus=], return false.
+          </li>
+          <li>
+            Let |focusedDocument| be the |topLevelBC|'s
+            <a href="https://html.spec.whatwg.org/multipage/interaction.html#currently-focused-area-of-a-top-level-browsing-context">
+              currently focused area</a>'s [=Node/node document=].
+          </li>
+          <li>
+            If |relevantGlobal|'s [=relevant settings object=]'s [=origin=] is [=same origin=] with
+            |focusedDocument|'s [=origin=], return true.
+          </li>
+          <li>
+            Otherwise, return false.
+          </li>
+        </ol>
+      </p>
+      <p>
+        The <dfn>passes privacy test</dfn> steps given the argument |observer:PressureObserver| and
+        its [=relevant global object=] |relevantGlobal|, are as follows:
+        <ol>
+          <li>
+            If |relevantGlobal| is a {{WorkerGlobalScope}} object:
+            <ol>
+              <li>
+                If |relevantGlobal|'s relevant worker is not a <a href="https://html.spec.whatwg.org/multipage/workers.html#active-needed-worker">
```

Comment:
This is not introduced in this PR (indentation changed so that is why it shows up) - Do you mind filing a bug about this?

I guess it should be "associated worker". The specs says

```
10.2.1 The global scope
The global scope is the "inside" of a worker.
```

Which is also very hand wavy :-) There doesn't seem to be a way to refer to the "outside". Would "associated" be OK? Or do you have better suggestions?

Reply:
Yes, this actually seems unneeded now, because we do this check directly instead of relying on the "active needed worker" semi algorithm. - Let me remove it, nice catch!